src/_locales/en_US/messages.json

```diff
@@ -359,6 +359,22 @@
         "message": "Privacy Policy",
         "description": "Shown at the bottom of the intro page."
     },
+    "intro_easy_to_manage": {
+        "message": "Easy to manage",
+        "description": "Intro page paragraph heading on Chrome."
+    },
+    "intro_easy_to_manage_paragraph": {
+        "message": "Pin this extension to your toolbar for easy access. The Privacy Badger icon will show you how many trackers it blocks on each page. Click on the icon to manage your privacy settings.",
+        "description": "Intro page paragraph on Chrome."
+    },
+    "intro_pin_action": {
+        "message": "Pin Privacy Badger to your toolbar",
+        "description": "Shown on arrow pointing at Chrome extension icon"
+    },
+    "intro_pin_purpose": {
+        "message": "for easy access to the \"Disable for this site\" button.",
```

Comment:
We shouldn't split up locale messages like this, as it makes it harder for translators to make sense of what they are translating, and there may be grammar and other locale difference that don't work well with the way we've chosen to split the message.

Let's use a single message with placeholders instead.

Reply:
I split the locale messages so that I could apply different font-families to each. I'll try doing the same thing with placeholders!